README.md

```diff
@@ -105,13 +141,13 @@ cd ~/Firmware
 git submodule update --init --recursive
 ```
 
-10. Install Firmware dependencies.
+2. Install Firmware dependencies.
 
 ```bash
 apt install libopencv-dev python-jinja2 protobuf-compiler
```

Comment:
Is this really everything. We list a heap more in http://dev.px4.io/en/setup/dev_env_linux_ubuntu.html#common-dependencies

It would be "nice" if we could somehow express this setup as a delta against http://dev.px4.io/en/setup/dev_env_linux.html#gazebo-with-ros